recipes/ssht/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.12)
+project(PackageTest C)
+
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
+find_package(ssht)
+add_executable(test_package example.c)
+target_link_libraries(test_package ssht::ssht)
+target_compile_features(test_package PUBLIC c_std_99)
+set_target_properties(test_package PROPERTIES RUNTIME_OUTPUT_DIRECTORY
+                                              "${PROJECT_BINARY_DIR}/bin")
```

then the rest is handled by our template... `conan_basic_setup()` does the magic

```suggestion
cmake_minimum_required(VERSION 3.1)
project(test_package CXX)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup()

find_package(Ssht REQUIRED)

add_executable(${PROJECT_NAME} example.c)
target_compile_features(${PROJECT_NAME} PUBLIC c_std_99)
target_link_libraries(${PROJECT_NAME} ssht::ssht)
```